docs/cargo_features.md

```diff
@@ -31,6 +31,7 @@ The default feature set enables most of the expected features of a game engine,
 |bevy_ui|A custom ECS-driven UI framework|
 |bevy_winit|winit window and input backend|
 |default_font|Include a default font, containing only ASCII characters, at the cost of a 20kB binary size increase|
+|fixed_time|Provides the fixed update schedules|
```

Comment:
Maybe this should link somewhere on our docs so that new users encountering this flag don't get confused what any of that means? Could also link to the fixed timestep example, that one has a nice documentation of the feature.

Reply:
Should we just include the link in the description?